mojave/items/storage/radiopack.dm

```diff
@@ -1,4 +1,4 @@
-/obj/item/ms13/radiopack
+/obj/item/ms13/storage/backpack/radiopack
```

Comment:
Pathing should be changed for this, ideally with the other backpacks as well if you're willing.